plugins/dependencies/src/main/kotlin/dependencies/Testing.kt

```diff
@@ -88,8 +88,13 @@ object Testing {
             const val piTest = "$artifactPrefix-pitest:_"
         }
 
+        @Suppress("DEPRECATION")
         val extensions = Extensions
```

Comment:
The deprecated annotation should also be on this variable extensions

Reply:
Added ✌🏼